[Windows] MinGW within Nim install is missing libraries

``` [Linking]
C:/Nim/dist/mingw/bin/../lib/gcc/x86_64-w64-mingw32/4.9.1/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lgdi32
C:/Nim/dist/mingw/bin/../lib/gcc/x86_64-w64-mingw32/4.9.1/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lcomdlg32
Error: execution of an external program failed
```

Just want to let you know that trying to compile with _--app:gui_ param results to this.
Seems like currently the compiler comes with mingw thats missing libs
(Its minor stuff and i feel like whiner )

Windows 7 , nim-0.11.2_x64.exe

--- a/tools/trimcc.nim
+++ b/tools/trimcc.nim
@@ -10,7 +10,7 @@ fenv.h fmtmsg.h fnmatch.h ftw.h errno.h glob.h gtmath.h if.h in.h ipc.h
 langinfo.h locale.h math.h mman.h netdb.h nl_types.h poll.h pthread.h pwd.h
 sched.h select.h semaphore.h signal.h socket.h spawn.h stat.h statvfs.h stdio.h
 stdlib.h string.h strings.h tcp.h time.h types.h ucontext.h uio.h utsname.h
-unistd.h wait.h varargs.h windows.h zlib.h
+unistd.h wait.h varargs.h windows.h zlib.h libgdi32.a libcomdlg32.a
 """.split
 
 proc includes(headerpath, headerfile: string, whitelist: StringTableRef) =

